comm/src/main/scala/coop/rchain/comm/transport/buffer/ConcurrentQueue.scala

```diff
@@ -21,7 +21,7 @@ abstract class ConcurrentQueue[A] {
   def drain(buffer: mutable.Buffer[A], limit: Int): Unit
 }
 
-@SuppressWarnings(Array("org.wartremover.warts.Return"))
+@SuppressWarnings(Array("org.wartremover.warts.Return", "org.wartremover.warts.Var"))
```

Comment:
the `var` is only used once, in `drain` - move the suppression there?

Reply:
`ConcurrentQueue` is copy/paste from monix - in thi case I thought it is a good idea to keep all supresswarning upfront showing that sooner we move away from monix queues the better ;)